packages/rum/src/boot/recorderApi.ts

```diff
@@ -164,6 +164,9 @@ function isBrowserSupported() {
     // Array.from is a bit less supported by browsers than CSSSupportsRule, but has higher chances
     // to be polyfilled. Test for both to be more confident. We could add more things if we find out
     // this test is not sufficient.
-    typeof Array.from === 'function' && typeof CSSSupportsRule === 'function' && 'forEach' in NodeList.prototype
+    typeof Array.from === 'function' &&
+    typeof CSSSupportsRule === 'function' &&
+    typeof URL.createObjectURL === 'function' &&
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏we could do`typeof getSupportedUrl()?.createObjectURL === 'function'` and use `getSupportedUrl()!.createObjectURL` when building the Worker, no?